src/Arcus.Security.AzureFunctions/Arcus.Security.AzureFunctions.csproj

```
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <Authors>Arcus</Authors>
+    <Company>Arcus</Company>
+    <Description>Contains Azure Functions core functionality for Arcus.Security</Description>
```

Maybe better since it's a phrase?
```suggestion
    <Description>Contains Azure Functions core functionality for Arcus Security</Description>
```